internal/services/videoindexer/video_indexer_account_resource.go

```diff
@@ -76,6 +78,13 @@ func (r AccountResource) Arguments() map[string]*pluginsdk.Schema {
 
 		"identity": commonschema.SystemAssignedUserAssignedIdentityRequired(),
 
+		"public_network_access": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Default:      string(accounts.PublicNetworkAccessEnabled),
+			ValidateFunc: validation.StringInSlice(accounts.PossibleValuesForPublicNetworkAccess(), true),
```

Comment:
Was there a specific reason for validating this insensitively? We'd prefer to use case-sensitive forms where possible

```suggestion
			ValidateFunc: validation.StringInSlice(accounts.PossibleValuesForPublicNetworkAccess(), false),
```

Reply:
Fixed.